viewer/Viewer.h

```diff
@@ -111,23 +124,58 @@ namespace Enki
 		typedef QMapIterator<const std::type_info*, const std::type_info*> ManagedObjectsAliasesMapIterator;
 		ManagedObjectsAliasesMap managedObjectsAliases;
 		
+		typedef std::pair<QString, unsigned int> viewerMessage;
+		std::list<viewerMessage> messageList;
+		QString controlError1;	// dislpayed if user try to move an object in trackball mode : action forbiden to prevent glitchies
+		QString controlError2;	// dislpayed if user try to move camera in trackball mode
+		QString controlHelp;	// dislpayed if user press F1
+
+		struct ExtendedAttributes
+		{
+			bool movableByPicking;
+
+			ExtendedAttributes():movableByPicking(false){};
+		};
+		std::map<PhysicalObject*, ExtendedAttributes> objectExtendedAttributesList;
```

Comment:
ti's simplest:
 - a second attribute class assigned to an object by pointer (as UserData) need a new pointer for all PhysicalObject only used by the viewer and accessible anywhere. But we don't need it for all, and accessibility is just for the viewer.
 - in term of performance it's not too bad: we check the pickability only in a object moving action, and the complexity log and relative to the objectExtendedAttributesList, so for 20 object in the aseba playground R2T2 case (worth for me)